packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -14,7 +14,7 @@ import type {
 import { StaticIntervalPollingController } from '@metamask/polling-controller';
 import { Mutex } from 'async-mutex';
 
-import { fetchExchangeRate as defaultFetchExchangeRate } from './crypto-compare';
```

Comment:
I wonder if this `crypto-compare` should be a in `services/` folder instead?

Reply:
Possibly, but I'd classify that change as out of scope for this PR since it would also imply changing the location of the Price API service